### ISSUE TITLE
Corner radius mixins for text field don't seem to work

### ISSUE THREAD

**Issue by Ranguna** [material-components/material-components-web-react#162]
Hello again,

If I use any of the following mixins:
``` scss
mdc-notched-outline-corner-radius();
mdc-notched-outline-idle-corner-radius();
mdc-text-field-box-corner-radius();
mdc-text-field-textarea-corner-radius();
```
With any type of text field, I expect the corner radius to change but it stays the same.

Is this a problem on my end ?

___
Also, I found another bug with text fields where the outlined (none idle) border is placed in the middle of the actual text field when it inherits a `text-align: center` style from a parent (and maybe even if it's set to itself), like this:
![image](https://user-images.githubusercontent.com/5003755/43039560-e4a6a3f0-8d27-11e8-9880-3853d6b6ba86.png)
I fixed it by setting `text-align` to `start` for the `mdc-notched-outline` class:
``` scss
.mdc-notched-outline {
  text-align: start;
}
```
Should I open a new issue ?

Thanks :+1:

**Comment by bonniezhou** [comment]
Hi @Ranguna, are you importing the correct Sass file: `@material/react-text-field/index.scss`? The `mdc-text-field-*-corner-radius` seems to work for me. Note that the mixin should only work for its corresponding variant (e.g. the `mdc-text-field-box-corner-radius` should only work for the box text field).

Looks like the `mdc-text-field-outline-corner-radius` mixin is missing, so I filed [this issue](https://github.com/material-components/material-components-web/issues/3172) in the MDC Web repo.

Thanks for catching that text-align bug! Feel free to send a PR to set `text-align: start;` 🙂

**Comment by Ranguna** [comment]
Hah I wasn't importing that Sass file.. And I was calling that mixin with a string `"20px"` instead of `20px`.
My problem has been fixed, thank you :ok_hand: 

Yeah I noticed `mdc-text-field-outline-corner-radius` was missing and I had to use both notched mixins to get the same effect but now that it's working, the floating lable isn't being positioned to the right spot:
![image](https://user-images.githubusercontent.com/5003755/43071025-5d4a4b08-8e69-11e8-92b0-f415e1bb4c0f.png)
The notch is being set nicely but the lable doesn't move accordingly, that's probably because I'm using `notched-outline` mixins to change the radius and thus the lable will stay in the same place because nothing was done to move it to the right position after the updated corner.
I believe that'll be fixed once your issue has been resolved since a mixin for the outlined textfield will take all this into account.
___
A PR to [material-components-web](https://github.com/material-components/material-components-web) ?
Somewhere along [this mixin](https://github.com/material-components/material-components-web/blob/master/packages/mdc-textfield/_mixins.scss#L257) ?
I haven't read any guidelines on updating you code so I'm not really sure where to fit that `text-align: start;` into.
